ringtail/ringtailcore.py

```diff
@@ -896,32 +994,39 @@ def add_results_from_vina_string(self,
             OptionError
         """
         # Method currently only works with vina output, set automatically
-        if self.docking_mode != "vina": self.docking_mode = "vina"
+        if self.docking_mode != "vina":
```

Comment:
I'm guessing this is here to prepare for some sort of future implementation where more than just vina works?

Reply:
Yes, for example I believe the new autodock will provide result strings over files